src/test/java/org/gridsuite/study/server/NetworkModificationTreeTest.java

```diff
@@ -694,5 +701,36 @@ private void assertModificationNodeEquals(AbstractNode expected, AbstractNode cu
         assertEquals(expectedModificationNode.getSecurityAnalysisResultUuid(), currentModificationNode.getSecurityAnalysisResultUuid());
         assertEquals(expectedModificationNode.getBuildStatus(), currentModificationNode.getBuildStatus());
     }
+
+    @Test
+    public void testNodeName() throws IOException {
+        RootNode root = createRoot();
+        String nodeName = "node 1";
+        final NetworkModificationNode node = buildNetworkModification(nodeName, "potamus", null, VARIANT_ID, LoadFlowStatus.RUNNING, loadFlowResult2, UUID.randomUUID(), BuildStatus.BUILT);
+        createNode(root.getStudyId(), root, node);
+        node.setId(null);
```

Comment:
Maybe create a `createNodeFail` method here which could be reused in other tests